index-cs.html

```diff
@@ -0,0 +1,748 @@
+        <br/><br/>
+        A naopak lidé bez roušky se častěji budou vídat s dalšími lidmi bez roušek.
+        Je to důsledkem tendence mít ve svém okolí podobně smýšlející lidi s podobnými názory a pohledy. Tento jev <a href="https://www.sciencedirect.com/science/article/pii/S2468042720300191">snižuje</a> ochrannou sílu roušek a respirátorů, a to především tím, že lidé bez ochrany obličeje <a href="https://github.com/aatishb/maskmath/blob/master/model/mathmodel.ipynb">jsou o něco méně chráněni</a> rouškami a respirátory těch, kteří je nosí. 
+        
+        <br/><br/>
+        Jak <a href="https://www.sciencedirect.com/science/article/pii/S2468042720300191">ve své analýze</a> píší David Fisman, Amy Greer a Ashleigh Tuite, "dopad nošení roušek anebo jiné ochrany obličeje na šíření infekce COVID-19 bude největší, pokud bude zajištěna jejich dostupnost i pro znevýhodněné skupiny obyvatelstva". 
+      </span>
+    </aside>
+
+    <h2>Jakou cenu platíme</h2>
+
+    <p>
+      Pojďme se na sílu roušek a respirátorů podívat z jiného úhlu.
+      Roušky a respirátory zachraňují lidské životy. Snižují totiž počet nakažených, což snižuje počet lidí, kteří během pandemie zemřou.
+    </p>
+
+    <p>
+      Čím více lidí nosí roušky a respirátory, tím více klesá reprodukční číslo R0.
+      S poklesem R0 klesá i počet nakažených lidí.
+      Namísto vizualizace R0 si tedy můžeme ukázat, jak nošení ochrany obličeje pohne <i>podílem nakažených lidí</i> v populaci.
+      
+      
+    </p>
+
+    <p>
+      S využitím často používaného matematického modelu (<a href="https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology">SIR model</a>) můžeme spočítat, jak se číslo R0 projeví na počtu nakažených lidí. 
+      (O modelech SIR se více dozvíte v tomto výborně zpracovaném <a href="https://www.youtube.com/watch?v=gxAaO2rsdIs">videu</a>.)
+    </p>
+    
+    <p>
+      Tento model výrazně zjednodušuje (předpokládá náhodné potkávání lidí a nulová opatření). Ukazuje nám ale přibližný odhad 
+      toho, kolik lidí může nošení roušek a respirátorů ochránit před nákazou. A naopak, kolik lidí se nakazí, když roušky/respirátory nikdo nenosí. </p>
+
+  </div>
+
+  <div class="graphic">
+    <div class="graphic-container">
+
+      <label class="row half bigsplash">
+        <div>R0: <span class="dotted">{{R0}}</span></div>
+        <slider v-model="R0" :min="1" :max="3" :step="0.01"></slider>
+      </label>
+
+      <label class="row half bigsplash">
+        <div>Účinnost ochrany (výdech): <span class="dotted">{{convertToPercent(eout)}}</span>%</div>
+        <slider v-model="eout" :min="0" :max="0.99" :step="0.01"></slider>
+      </label>
+
+      <label class="row half bigsplash">
+        <div>Účinnost ochrany (nádech): <span class="dotted">{{convertToPercent(ein)}}</span>%</div>
+        <slider v-model="ein" :min="0" :max="0.99" :step="0.01"></slider>
+      </label>
+
+      <graph :traces="graph3Traces(
+        '% Infected'

```

Comment:
@progproc we added a new line of code here so that this hover text can be translated. Could you please translate this phrase?